### PR TITLE
PUB-1064 - Create Media Account Review Screen

### PR DESCRIPTION
**Pull request by ChrisS1512**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1064

### Change description ###

This is the first step of the media-account-review process, which lets users approve / reject the application before passing it on to the further screens

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by furnivall — APPROVED** [review]
Looks good to me!

**Review by joshblackmoor — DISMISSED** [review]

File: src/main/routes/routes.ts [hunk 1]
```diff
@@ -141,6 +141,10 @@ export default function(app: Application): void {
   app.post('/subscription-urn-search', ensureAuthenticated, app.locals.container.cradle.subscriptionUrnSearchController.post);
   app.get('/subscription-urn-search-results', ensureAuthenticated, app.locals.container.cradle.subscriptionUrnSearchResultController.get);
   app.post('/unsubscribe-confirmation', ensureAuthenticated, app.locals.container.cradle.unsubscribeConfirmationController.post);
+  app.get('/media-account-review', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.get);
+  app.get('/media-account-review/image', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.getImage);
+  app.post('/media-account-review/approve', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.approve);
+  app.post('/media-account-review/reject', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.reject);
```
Comment (kurt-uk) [inline src/main/routes/routes.ts:147]: these are restricted admin paths 👇🏼 👇🏼 👇🏼

**Review by kurt-uk — CHANGES_REQUESTED** [review]

File: src/main/routes/routes.ts (hunk 1 above)
Comment (ChrisS1512) [reply src/main/routes/routes.ts]: Moved these down into the restricted paths section

**Review by furnivall — APPROVED** [review]

File: src/main/views/media-account-review.njk [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      [
+        {
+          text: name,
+          classes: 'govuk-!-width-one-third'
+        },
+        {
+          text: applicantData.fullName
+        },
+        {
+          text: ""
```
[the remark this answers is not in the document]
Comment (ChrisS1512) [reply src/main/views/media-account-review.njk]: Nice, updated this now and added some tests

**Review by kurt-uk — DISMISSED** [review]

**Review by furnivall — DISMISSED** [review]

**Review by furnivall — APPROVED** [review]
lgtm

**Review by joshblackmoor — APPROVED** [review]

**Pull request merged**